src/moonlight-server/events/events.hpp

```diff
@@ -192,6 +192,16 @@ struct LeaveLobbyEvent {
   const std::size_t moonlight_session_id;
 };
 
+struct PauseLobbyEvent {
+  const std::string lobby_id;
+  std::optional<std::vector<short>> pin = std::nullopt;
+};
+
+struct ResumeLobbyEvent {
+  const std::string lobby_id;
+  std::optional<std::vector<short>> pin = std::nullopt;
+};
```

Comment:
I think the names here are misleading, I'd use something like `PauseRunner` and `ResumeRunner` instead. In fact, with your implementation, when the pause gets called the lobby is effectively destroyed (since everyone leaves).